vendor/engines/ldap_authentication/ldap_authentication.gemspec

```diff
@@ -16,6 +16,6 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "README.md"]
   s.test_files = Dir["spec/**/*"]
 
-  s.add_dependency "rails", "~> 4.2.7.1"
+  s.add_dependency "rails", "~> 4.2.9"
```

Comment:
I'm wondering if we should make the engine rails dependencies looser. So something like `~> 4.2`, or even `>= 4.2`. I just ran into this on DC while merging open. Or should we get rid of the dependency altogether?